src/nvim/ex_cmds_defs.h

```diff
@@ -71,6 +71,14 @@
 #define WORD1   (EXTRA | NOSPC) /* one extra word allowed */
 #define FILE1   (FILES | NOSPC) /* 1 file allowed, defaults to current file */
 
+// values for cmd_addr_type 
```

Comment:
Of course there is trailing white space :0 I'll fix this once I get home
later today.